tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,417 @@
+"""
+Provide tests for command line interface's get list of batches command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_list_batches_with_all_parameters(mocker):
```

Comment:
Why are you using a mocker in this test? Please, send the command with all parameters to the real test node.

Reply:
Done.